Brand new project won't run tests

```
> hubot-script@0.0.0 test /Users/eduardoaleixo/translucent/test/hubot-script-babel
> istanbul cover node_modules/mocha/bin/_mocha -- -R spec --compilers js:babel-core/register

No coverage information was collected, exit without writing coverage information
module.js:471
    throw err;
    ^

Error: Cannot find module 'hubot'
```

https://github.com/github/hubot/issues/1138

It seems that as hubot is distributed as coffescript only, we gotta transpile it to js.


Will provide a PR with a [fix](https://github.com/mtsmfm/hubot-test-helper/issues/33#issuecomment-229656357) in a few.

--- a/generators/app/index.js
+++ b/generators/app/index.js
@@ -67,6 +67,7 @@ module.exports = yeoman.generators.Base.extend({
 
 				self.copy('src/index.js', 'src/index.js');
 				self.copy('test/index.js', 'test/index.js');
+				self.copy('test/mocha.opts', 'test/mocha.opts');
 
 				try {
 					// Try copy the file if it exists, else catch the error silently

--- a/generators/app/templates/package.json
+++ b/generators/app/templates/package.json
@@ -23,10 +23,11 @@
     "babel-core": "^6.2.1",
     "babel-preset-es2015": "^6.1.18",
     "chai": "^3.4.1",
+    "coffee-script": "^1.12.4",    
   	"coveralls": "^2.11.6",
     "hubot": "^2.18.0",
-	"hubot-test-helper": "^1.4.4",
-	"istanbul": "^1.0.0-alpha.2",
+	  "hubot-test-helper": "^1.4.4",
+	  "istanbul": "^1.0.0-alpha.2",
     "mocha": "^2.3.4"
   }
 }

--- a/generators/app/templates/test/mocha.opts
+++ b/generators/app/templates/test/mocha.opts
@@ -1,0 +1,2 @@
+--require coffee-script/register
+--compilers coffee:coffee-script

--- a/package.json
+++ b/package.json
@@ -13,9 +13,9 @@
   "main": "generators\\index.js",
   "keywords": [
     "yeoman-generator",
-	"hubot",
-	"babel",
-	"es2015"
+    "hubot",
+    "babel",
+    "es2015"
   ],
   "dependencies": {
     "bluebird": "^3.0.6",
@@ -25,6 +25,7 @@
     "object-assign": "^4.0.1"
   },
   "devDependencies": {
+    "coffee-script": "^1.12.4",
     "coveralls": "^2.11.6",
     "eslint-config-degree53-core": "^1.0.6",
     "gulp": "^3.6.0",

--- a/package.json
+++ b/package.json
@@ -25,7 +25,6 @@
     "object-assign": "^4.0.1"
   },
   "devDependencies": {
-    "coffee-script": "^1.12.4",
     "coveralls": "^2.11.6",
     "eslint-config-degree53-core": "^1.0.6",
     "gulp": "^3.6.0",

--- a/test/app.js
+++ b/test/app.js
@@ -21,6 +21,7 @@ describe('generator-hubot-script-babel:app', function() {
 		assert.file([
 			'package.json',
 			'test/index.js',
+			'test/mocha.opts',
 			'src/index.js',
 			'.gitignore',
 			'.npmignore',

